extensions/PhabBugz/lib/Feed.pm

```diff
@@ -483,6 +483,8 @@ sub process_revision_change {
     }
 
     $attachment->update($timestamp);
+
+    undef $attachment;
```

Comment:
this shouldn't be required, but doesn't hurt either.

Reply:
Yeah that was from some previous troubleshooting and was not removed. Will remove.